app/javascript/app/components/result-card/result-card-component.jsx

```diff
@@ -13,19 +18,28 @@ const ResultCard = props => {
       </div>
       {result.matches &&
         result.matches.map(match => (
-          <div
+          <NavLink
             key={match.fragment}
-            id={match.idx}
+            to={`/ndcs/country/${result.iso_code3}/full?search=${query}&idx=0`}
             className={styles.match}
-            dangerouslySetInnerHTML={{ __html: match.fragment }}
-          />
+          >
+            <div
+              className={styles.text}
+              id={match.idx}
+              dangerouslySetInnerHTML={{ __html: match.fragment }}
```

Comment:
Could you highlight the fragment with the highlighted part?
![image](https://user-images.githubusercontent.com/10500650/30324766-738726a0-97c2-11e7-8021-e9e7640273b5.png)

Reply:
Well spotted! I forgot about this thanks 👀